worked_examples/annotation/how-to-annotate-patches.ipynb

```diff
@@ -0,0 +1,355 @@
+    "    download_url=\"https://mapseries-tilesets.s3.amazonaws.com/1inch_2nd_ed/{z}/{x}/{y}.png\",\n",
+    ")\n",
+    "\n",
+    "my_ts.query_map_sheets_by_wfs_ids(439)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set zoom level\n",
+    "my_ts.get_grid_bb()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download\n",
+    "my_ts.download_map_sheets_by_queries(path_save=\"./download/maps\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Patchify!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First load all the files\n",
+    "my_files = loader(\"./download/maps/*.png\")\n",
+    "my_files.add_metadata(metadata=\"./download/maps/metadata.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Then patchify!\n",
+    "my_files.patchify_all(path_save=\"./download/patches\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parent_df, patch_df = my_files.convert_images(save=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "patch_df.head()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preview the map\n",
+    "\n",
+    "Let's have a quick look at our map."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_files.show_sample(num_samples=1)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Annotate patches\n",
+    "\n",
+    "Now, we're ready to annotate patches. Let's set up an `Annotator`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mapreader import Annotator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "annotator = Annotator(\n",
+    "    patch_paths=\"./download/patches/*.png\",\n",
+    "    parent_paths=\"./download/maps/*.png\",\n",
+    "    metadata_path=\"./download/maps/metadata.csv\",\n",
+    "    delimiter=\",\",\n",
+    "    labels=[\"label 1\", \"label 2\"],\n",
+    "    username=\"rosie\",\n",
+    "    sortby=\"mean_pixel_R\",\n",
+    "    ascending=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
So the patch image is so small here. Is it possible to blow it up or make it zoomable? 
<img width="1035" alt="Screenshot 2023-12-07 at 15 46 32" src="https://github.com/Living-with-machines/MapReader/assets/20363927/8a9daac0-ac45-4b2c-b58d-b82fbc91b58e">

Reply:
Have just looked into this, I think its possible to make it bigger but hard to do without rescaling the image. 
Maybe this can be a follow on ticket future features of the annotator?

Reply:
Ah. Ok - this I think is very, very important, but OK to separate for future ticket.

How did it work in the previous annotator? I remember it being possible to see the patch as a pretty large image.

Reply:
Have just added this feature in, you can now pass ``resize_to`` kwarg to resize your patch (e.g. ``resize_to=300`` will give scale up a 100x100 pixel patch to 300x300 pixels. 

Will add docs for this now.

Reply:
Amazing. Thanks!